extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetColumnLocation.java

```diff
@@ -153,22 +154,19 @@ public <METADATA_TYPE> METADATA_TYPE getMetadata(
         if (!hasGroupingTable) {
             return null;
         }
-
-        final Function<String, String> defaultGroupingFilenameByColumnName =
-                ParquetTools.defaultGroupingFileName(tl().getParquetFile().getAbsolutePath());
+        final File parquetFile = tl().getParquetFile();
         try {
-            final GroupingColumnInfo groupingColumnInfo =
-                    tl().getGroupingColumns().get(parquetColumnName);
+            final GroupingColumnInfo groupingColumnInfo = tl().getGroupingColumns().get(parquetColumnName);
+            final String groupingFileRelativePath = groupingColumnInfo == null
+                    ? ParquetTools.getRelativeGroupingFilePath(parquetFile, parquetColumnName)
+                    : groupingColumnInfo.groupingTablePath();
```

Comment:
Seems to me like we should fall back to the old default path if we don't find explicit info or the new default path.